### PR TITLE
WIP: Miscellaneous improvements

### PR DESCRIPTION
**Pull request by Nathan-Bolte**
For all users: added idl_validname to read_hdf5 to prevent bad structure tags
For prefida in Windows: 	added test of OS to get_version.pro for doing prefida in Windows machine.

### CODE REVIEW

File: lib/read_hdf5.pro [hunk 1]
```diff
@@ -18,6 +18,9 @@ FUNCTION valid_name, name, bad_names=bad_names, post = post
         valid_name = name
     endelse
 
+    ; added 2016-09-15 by NGB to fix invalid structure tags
+    valid_name = idl_validname(valid_name, /convert_all)
+
```
Comment (lstagner) [inline lib/read_hdf5.pro]: What was the invalid structure tag you ran into?

File: lib/get_version.pro [hunk 2]
```diff
@@ -13,7 +13,11 @@ FUNCTION get_version, fidasim_dir
 
     version = ''
     git_dir = fidasim_dir+'/.git'
-    spawn,'command -v git ',git_command,/sh
+    if !VERSION.OS_FAMILY eq 'Windows' then begin
```
Comment (lstagner) [inline lib/get_version.pro]: You should probably use
```idl
if strcmp(!VERSION.OS_FAMILY ,'windows', /fold_case) then begin 
```
instead of `eq`

**Comment by lstagner** [comment]
Thanks for these. Once you think you are done reply in this thread and I'll do a final review and if there aren't any changes that need to be made I'll merge.

**Comment by Nathan-Bolte** [comment]
The tag had a dash "-" in it.


Nathan Bolte

**Comment by Nathan-Bolte** [comment]
I'm done with these changes. I made your suggested change on my copy but
will not do another pull request for that change.


Nathan Bolte

**Comment by lstagner** [comment]
Merged. By pushing your change to your branch it automatically updates the pull request so I have the changes you made to `get_version`.

**Pull request merged**